types/events.go

```diff
@@ -12,6 +12,7 @@ type Event struct {
 // stats is the runc specific stats structure for stability when encoding and decoding stats.
 type Stats struct {
 	CPU               Cpu                 `json:"cpu"`
+	Cpuset            Cpuset              `json:"cpuset"`
```

Comment:
should probably be "CPUSet" ?

```suggestion
	CPUSet            CPUSet              `json:"cpuset"`
```